### DISCUSSION TITLE
Node Unschedulable in longhorm, help me , please !!!

### DISCUSSION THREAD

**Discussion by Pedrocho** [longhorn/longhorn#2685] [Q&A] [answered]
HI everyones, 
 I have a kubernetes cluster with 3 nodes. 1 of them I have it in Unschedulable state and when I go to see the description of the node. I come across this.
 message: the disk default-disk-99adeb02898c9969(/var/lib/longhorn/) on
              the node server02 has 63176876032 available, but requires reserved
              94811403059, minimal 25% to schedule more replicas
 but , the pvc replicas are in Healthy mode.

How can i solve this problem ?

Problem -> node in state "Unschedulable" 
scheduledReplica:
          pvc-07x: 42949672960
          pvc-0ax: 53687091200
        storageAvailable:    63176876032
        storageMaximum:   315989229568
        storageScheduled:  96636764160

Many thanks

**Comment by joshimoo** [comment]
You can lower the reserve percentage if you want to store additional data on the device.
You should also change the overprovisioning percentage to 100% instead of the default 200%
It's best to use a dedicated disk for longhorn replica storage instead of storing them on the root disk, which is used by the rest of the OS.

```
pvc-07x:    42949672960 // 
pvc-0ax:    53687091200 //
Available:  63176876032 //  58GB
  Maximum: 315989229568 // 294GB
Scheduled:  96636764160 //  90GB

Reserved : Maximum(294) * Reserve(0.25) = 73GB
```

**Reply by Pedrocho** [reply]
hi joshimoo,

First of all, thanks for the reply. So in my case. Would have to change:

Storage Over Provisioning Percentage -> 100%
Storage Reserved -> 73 GB 
Storage Minimal Available Percentage -> 25%

and with this change it would disappear -> Unschedulable ?


Many thanks.

**Comment by joshimoo** [comment, marked as answer]
Currently your Storage Reserved : Maximum(294) * Reserve(0.25) = 73GB is higher then your available storage that's why the node becomes unschedulable. If you lower the `Storage Minimal Available Percentage` then your reserved storage goes down, and Storage Available will be bigger then Storage Reserved.

Example for 10% Storage Minimal Available Percentage:
Maximum(294) * Reserve(0.10) = 29.4GB

So now your Storage Available > Storage Reserved and you can schedule additional volumes.

**Comment by Pedrocho** [comment]
Many many thanks  for your aswers.

 but it may happen that it was a bug, I have my suspicions
